src/main/main.ts

```diff
@@ -78,5 +79,17 @@ function processNewRow(row: Row): void {
 }
 
 function deleteOldRows(processedRowIds: string[]): void {
-    // Will be implemented as part of https://3.basecamp.com/4474129/buckets/38736474/todos/7762398829
+    const propertyStore: DocumentProperties = getAllDocumentProperties();
+
+    for(const rowId in propertyStore) {
+        if(!processedRowIds.includes(rowId)) {
+
+            let rowBasecampMapping = propertyStore[rowId];
+            let roleTodoIdMap = rowBasecampMapping.roleTodoIdMap;
+            let todoIds = Object.values(roleTodoIdMap);
```

Comment:
```suggestion
            const rowBasecampMapping: RowBasecampMapping = propertyStore[rowId];
            const roleTodoIdMap: RoleTodoIdMap = rowBasecampMapping.roleTodoIdMap;
            const todoIds: string[] = Object.values(roleTodoIdMap);
```
Small change. Otherwise looks good 👍